app/services/container_dashboard_service.rb

```diff
@@ -1,6 +1,8 @@
 class ContainerDashboardService < DashboardService
   include ContainerServiceMixin
   include Mixins::CheckedIdMixin
+  include ActionView::Helpers::DateHelper
```

Comment:
This should not be necessary if `TextualMixins::TextualRefreshStatus` itself includes `ActionView::Helpers::DateHelper`